.github/workflows/build-and-test.yml

```diff
@@ -99,7 +98,7 @@ jobs:
       - name: Upload built charm
         uses: actions/upload-artifact@v4
         with:
-          name: charm-artifact-${{ matrix.part }}
+          name: charm-${{ matrix.part }}
```

Comment:
This seems to have reverted to the old version. Was it deliberate or just a side-effect of rebasing?